code/modules/events/wizard/madness.dm

```diff
@@ -0,0 +1,28 @@
+/datum/round_event_control/wizard/madness
+	name = "Curse of Madness"
+	weight = 1
+	typepath = /datum/round_event/wizard/madness
+	earliest_start = 0 MINUTES
+
+	var/forced_secret
+
+/datum/round_event_control/wizard/madness/admin_setup()
+	if(!check_rights(R_FUN))
+		return
+
+	var/suggested = pick(strings("redpill.json", "redpill_questions"))
```

Comment:
String duplication, these should be defined somewhere

Reply:
Also, given that the second parameter is `directory` how does this even work? Does the original?

Reply:
read the code